modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -33,23 +33,23 @@ Use the following values to configure the EDIFACT UNB identifier type:
 Select the qualifier types for the partner’s EDIFACT UNB identifier from the drop-down menu.
 |Yes
 | Identifier value
-| Specify the EDIFACT-UNB ID for your trading partner, which can be up to 15 characters. Your trading partner supplies you with the proper EDIFACT Interchange sender and receiver IDs during onboarding.
+| Specify the EDIFACT-UNB ID for your partner, which can be up to 15 characters. Your partner supplies you with the proper EDIFACT Interchange sender and receiver IDs during onboarding.
```

Comment:
Interchange init cap OK?

Reply:
On line 69, Sender and Receiver are also init cap.

Reply:
"Your partner supplies you with the" > "Your partner provides the"

Reply:
I see the capitalization both ways on Google results.  At some point, I will figure this out and will make a pass to make this consistent throughout the doc.